mesonbuild/modules/gnome.py

```diff
@@ -1170,11 +1171,26 @@ def abs_filenames(files: T.Iterable['FileOrString']) -> T.Iterator[str]:
                   '--modulename=' + modulename,
                   '--moduleversion=' + moduleversion,
                   '--mode=' + kwargs['mode']]
+        opt = mesonlib.OptionKey('gtkdoc', module='gnome')
+        feature = state.environment.coredata.options[opt]
+        if feature.is_disabled():
+            mlog.log(mlog.bold('gnome.gtkdoc()'), 'skipped: feature', mlog.bold('gnome.gtkdoc'), 'disabled')
+            return ModuleReturnValue(Disabler(), [])
```

Comment:
You can return Disabler() directly, ModuleReturnValue is only needed in the case you have build targets.